class-cf7-gutenberg-block-plugin.php

```diff
@@ -23,7 +23,7 @@ function gutenberg_enqueue_block_editor_assets() {
 		wp_enqueue_script(
 			'gutenberg-cf7',
 			plugins_url( '/block-esnext/block.build.js', __FILE__ ),
-			array( 'wp-blocks', 'wp-i18n', 'wp-element', 'underscore' ),
+			array( 'wp-blocks', 'wp-components', 'wp-editor', 'wp-i18n', 'wp-element', 'underscore' ),
```

Comment:
@HardeepAsrani curious to know about the `underscore` dependecy

Reply:
Don't know. It was there before, so didn't touch.